src/bindings/python/tests/test_graph/test_create_op.py

```diff
@@ -885,6 +886,37 @@ def test_roi_align(data_shape, rois, batch_indices, pooled_h, pooled_w, sampling
     assert list(node.get_output_shape(0)) == expected_shape
 
 
+@pytest.mark.parametrize(
+    ("data_shape", "rois", "batch_indices", "pooled_h", "pooled_w", "sampling_ratio", "spatial_scale", "clockwise_mode", "expected_shape"),
```

Comment:
(Detail, optional) The test is parametrized by the input shapes, that could be reflected in the test param names for a better readability.
```suggestion
    ("data_shape", "rois_shape", "batch_indices_shape", "pooled_h", "pooled_w", "sampling_ratio", "spatial_scale", "clockwise_mode", "expected_shape"),
```

Reply:
`test_roi_align` has similar arguments. Should I also modify those as well?
https://github.com/openvinotoolkit/openvino/blob/b0b36a6c28b4046f817025f427b24b6f30848951/src/bindings/python/tests/test_graph/test_create_op.py#L856-L857

Reply:
Seems like the naming could be slightly adjusted for both tests, but the PR is already good.
Let's wait for the green CI (validation cycle) and we will merge it (no more changes required).